komodo/build.py

```diff
@@ -42,7 +42,7 @@ def dfs(pkg, version, pkgs, repo):
     return flatten(dependencies)
 
 
-def rpm(pkg, ver, path, data, prefix, *args, **kwargs):
+def rpm(pkg, ver, path, prefix):
     # cpio always outputs to cwd, can't be overriden with switches
```

Comment:
Recommend adding docstring to this and all other functions. Might save the next person a bit of time figuring this module out.

Reply:
I think that is a separate pylint issue, but it is definitely on the to-do list!